tools/utilAppRunner.js

```diff
@@ -14,7 +14,7 @@ function runUtilApp (cmd, file, stdioOptions) {
   }
   cmd = cmd.split(' ')
   if (process.env.NODE_ENV === 'development') {
-    cmd.push('--user-data-dir=brave-development')
+    cmd.push('--user-data-dir=../../../brave-development')
```

Comment:
@willy-b - what's the motivation behind this change, i think it might break some other tests...